### ISSUE TITLE
Grave accent aka backtick (`) is a combining character and it shouldn’t be

### ISSUE THREAD

**Issue by UnaiM** [Avi-D-coder/fira-mono-italic#2]
I’m sure you’ll agree this is a **grave** issue :smirk:

When I write ``this is `a little` test`` it appears as ``this is à little`test``. [zwaldowski/Fira](/zwaldowski/Fira), [mozilla/Fira](/mozilla/Fira) or [bBoxType/FiraSans](/bBoxType/FiraSans) don’t show this issue.

**Comment by Avi-D-coder** [comment]
Does the issue occur on [FiraCode-italic](https://github.com/Avi-D-coder/FiraCode-italic)

**Comment by UnaiM** [comment]
Yes it does— plus I’ve just found out that both with [fira-mono-italic](/Avi-D-coder/fira-mono-italic) and [FiraCode-italic](/Avi-D-coder/FiraCode-italic) sometimes it combines with the letter behind it and some other times with the one ahead of it.

**Comment by Avi-D-coder** [comment]
what os, editor, etc are you on.

**Comment by UnaiM** [comment]
I’m on CentOS 8 GNOME, and it happens in gedit, Atom and Chrome. Terminal is unaffected (just found out now).

**Comment by Avi-D-coder** [comment]
I downloaded gedit on arch and I still can't reproduce the error.
